tests/integration/flink/test_flink_integration.py

```diff
@@ -3,11 +3,14 @@
 import tempfile
 import time
 from importlib.metadata import version
+from pathlib import Path
 
 import pytest
 import xarray as xr
 from packaging.version import parse as parse_version
 
+TEST_DATA_DIR = Path(__file__).parent.parent / "test-data"
```

Comment:
Looks pretty relative to me! Anyway, yeah, this is what a competent programmer would do